Sources/Sentry/include/SentryAppState.h

```diff
@@ -42,6 +43,8 @@ SENTRY_NO_INIT
 
 @property (nonatomic, assign) BOOL isANROngoing;
 
+@property (nonatomic, copy) NSNumber *_Nullable timezoneOffset;
```

Comment:
`m`: We use three different data types. In the constructor, `timezoneOffset` is `NSTimeInterval`, here it is `NSNumber` and `[NSTimeZone localTimeZone].secondsFromGMT]` is `NSInteger`. Can we please align these. I would suggest to use `NSInteger` everywhere and renaming this to `secondsFromGMT`.

Reply:
Ah right, the NSTimeInterval was from a different property I used for a while, that's easily changed to NSInteger.

But changing the NSNumber to NSInteger.. can NSInteger be null? I really need to refresh my knowledge of optionals in Obj-C 😅 I made it an NSNumber specifically because of the optionality.

Reply:
> and renaming this to secondsFromGMT

I think `timezoneOffset` is a lot more clear. `secondsFromGMT` doesn't really tell you that this about a timezone thing.